e2e/browser/testApp/package.json

```diff
@@ -16,30 +16,16 @@
     "start": "next start"
   },
   "dependencies": {
-    "@inrupt/prism-react-components": "^0.13.1-alpha.2",
     "@inrupt/solid-client": "^1.20.1",
     "@inrupt/solid-client-access-grants": "file:../../../",
     "@inrupt/solid-client-authn-browser": "^1.12.2",
-    "@inrupt/solid-ui-react": "^2.7.0",
-    "@material-ui/core": "^4.12.3",
-    "@solid/lit-prism-patterns": "^0.13.1-alpha.2",
-    "@solid/lit-prism-theme-sdk-default": "^0.13.1-alpha.2",
-    "clsx": "^1.1.1",
-    "jss": "^10.5.1",
-    "jss-preset-default": "^10.5.1",
-    "next": "^12.1.0",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0"
+    "next": "^13.0.4",
+    "react": "^18.2.0"
   },
   "devDependencies": {
-    "@inrupt/eslint-config-lib": "^0.4.1",
+    "@types/node": "^18.11.9",
+    "@types/react": "^18.0.25",
     "eslint-config-next": "^12.1.5",
-    "typescript": "^4.2.3"
-  },
-  "overrides": {
-    "next@^12.1.0": {
-      "react": "^16.14.0",
-      "react-dom": "^16.14.0"
-    }
+    "typescript": "^4.9.3"
```

Comment:
We might be best sticking to `"typescript": "^4.7.4"` until we do the upgrade task, then we can tackle them all at once and know that everything is on the same version of typescript

Reply:
e7121f40248b3042775ce554c9d9d18f5611ca32